content/guide/_index.md

```diff
@@ -8,145 +8,207 @@ Welcome! This guide will help you get started with a Hugo Theme, including how t
+```bash
+cd my-hugo-website
+git init
+```
 
- * **Source files** are files that are meant to be processed by a theme's build tools.
+### Step 4: Adding a theme
 
- * **Compiled files** are generated as a result of running a compiling process (also called a "build process") 
-on the source files.
+There are three ways to add a theme. Choose the one most convenient for you. 
 
- * **Static files** are ones that aren't processed or generated.
+#### Method 1: Install the theme by copying
 
-**Here are a few real-world examples of each type of file:**
+If you have a compressed (.zip, .tar or any other format) file with the theme then extract it and copy the resulting folder into the folder named `themes` in your repository. You should end up with a folder structure like the following:
 
- * **SCSS files always source files** because they must be compiled by your theme's build tool to generate a CSS file that a browser can understand. SCSS files are located in `assets` folder.
+```text
+|- my-hugo-site/
+   |- themes/
+      |- my-hugo-theme/
+         |- layouts/
+         |- theme.toml
+         |- other files and folders
+   |- other files and folders
+```
 
- * **HTML files with any non-standard syntax are source files.** Some themes make use of `partials`, `shortcodes`, `internal templates` to create elements. Those must be compiled by `Hugo` to generate standard HTML. Those HTML files are located in `layouts` folder.
+#### Method 2: Install the theme as Git submodule
 
- * **exampleSite folder has source files.** This folder includes all `content`, `data`, and `config` files which must be compiled to run on the browser.
+If you are using a Git repository for your website and the theme is available in a Git repository then you can install the theme as submodule.
 
- * **Fonts, images, and plugins are static files.** These don't require any processing to be used and aren't generated from any other source. Fonts and Plugins are located in `static` folder. And Images are located in `exampleSite/static` folder because images need to change when you build a real site.
+```bash
+cd my-hugo-website
+git submodule add git@github.com:themeauthorname/my-hugo-theme.git themes/my-hugo-theme
+git commit -m "add theme my-hugo-theme as submodule"
+```
 
- * **The output of those above examples are compiled files.** The CSS files are generated by processing the source SCSS. The HTML files are generated by processing the pseudo-HTML source. And all pages are generated from exampleSite contents. All are generated from source files and will be overwritten if the source is compiled again. Output files can be found on `exampleSite/public` folder, when you run `hugo --themesDir ../..` this command. If you want the minified output, then run `hugo --minify --themesDir ../..`.
+This will create a subfolder in themes/my-hugo-theme that contains the files for your theme. 
```

Comment:
Codacy found an issue: [[maximum-line-length] Line must be at most 80 characters](https://app.codacy.com/gh/gethugothemes/gethugothemes-docs/pullRequest?prid=6304190)